### PR TITLE
feat(Filmstrip) add logic command resize filmstrip

### PR DESCRIPTION
**Pull request by dudumanbogdan**
- add logic command resize filmstrip
- needs also this PR https://github.com/jitsi/jitsi-meet-spot/pull/922

### CODE REVIEW

File: modules/UI/UI.js [hunk 1]
```diff
@@ -236,6 +236,13 @@ UI.toggleFilmstrip = function() {
     APP.store.dispatch(setFilmstripVisible(!visible));
 };
 
+/**
+ * Resize film strip
+ */
+UI.resizeFilmStrip = function(width) {
```
Comment (saghul) [inline modules/UI/UI.js:242]: LEt's avoid adding more logic to this deprecated file. Please just dispatch the action inline in API.js

File: react/features/filmstrip/actionTypes.ts [hunk 2]
```diff
@@ -18,6 +18,15 @@ export const SET_FILMSTRIP_ENABLED = 'SET_FILMSTRIP_ENABLED';
...
 export const SET_FILMSTRIP_VISIBLE = 'SET_FILMSTRIP_VISIBLE';
 
+/**
+ * The type of (redux) action which enlarges the filmstrip.
+ *
+ * {
+ *     type: RESIZE_FILMSTRIP,
+ * }
+ */
+export const RESIZE_FILMSTRIP = 'RESIZE_FILMSTRIP';
```
Comment (saghul) [inline react/features/filmstrip/actionTypes.ts:28]: Style: alpha sort.

File: react/features/filmstrip/actions.any.js [hunk 3]
```diff
@@ -4,7 +4,8 @@ import {
     SET_FILMSTRIP_ENABLED,
     SET_FILMSTRIP_VISIBLE,
     SET_REMOTE_PARTICIPANTS,
-    SET_VISIBLE_REMOTE_PARTICIPANTS
+    SET_VISIBLE_REMOTE_PARTICIPANTS,
+    RESIZE_FILMSTRIP
```
Comment (saghul) [inline react/features/filmstrip/actions.any.js:8]: style: alpha sort

File: react/features/filmstrip/middleware.web.js [hunk 4]
```diff
@@ -26,7 +26,8 @@ import {
     REMOVE_STAGE_PARTICIPANT,
     SET_MAX_STAGE_PARTICIPANTS,
     SET_USER_FILMSTRIP_WIDTH,
-    TOGGLE_PIN_STAGE_PARTICIPANT
+    TOGGLE_PIN_STAGE_PARTICIPANT,
```
Comment (saghul) [inline react/features/filmstrip/middleware.web.js:29]: ditto

File: react/features/filmstrip/middleware.web.js [hunk 5]
```diff
@@ -142,6 +143,15 @@ MiddlewareRegistry.register(store => next => action => {
         VideoLayout.refreshLayout();
         break;
     }
+    case RESIZE_FILMSTRIP: {
+        const { width } = action;
+
+        if (width) {
+            store.dispatch(setFilmstripWidth(width));
```
Comment (saghul) [inline react/features/filmstrip/middleware.web.js:150]: Should we reset it if undefined? Shouldn't setting it to 0 hide it?

**Review by saghul — CHANGES_REQUESTED** [review]
Left some comments, PTAL.

File: react/features/filmstrip/actionTypes.ts (hunk 2 above)
Comment (dudumanbogdan) [reply react/features/filmstrip/actionTypes.ts:28]: This rule is not really applied always, eg, I checked this file and we don't have the alpha sort.   Will it make sense to have this in eslint? I think https://eslint.org/docs/latest/rules/sort-vars

Comment (saghul) [reply react/features/filmstrip/actionTypes.ts:28]: For these files and constants file sure.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Jenkins please test this please.

**Pull request merged**